scribejava-apis/src/test/java/com/github/scribejava/apis/examples/ImgurExample.java

```diff
@@ -1,46 +1,42 @@
 package com.github.scribejava.apis.examples;
 
-import java.util.Scanner;
+import com.github.scribejava.apis.ImgurApi;
 import com.github.scribejava.core.builder.ServiceBuilder;
-import com.github.scribejava.apis.ImgUrApi;
-import com.github.scribejava.core.model.OAuthRequest;
-import com.github.scribejava.core.model.Response;
-import com.github.scribejava.core.model.Token;
-import com.github.scribejava.core.model.Verb;
-import com.github.scribejava.core.model.Verifier;
+import com.github.scribejava.core.model.*;
```

Comment:
It's better not to use star imports.
I've merged this pull request. I will remove star import in the next commit to the master